pyiron_continuum/schroedinger/schroedinger.py

```diff
@@ -157,12 +161,11 @@ def _hamiltonian(self, psi):
             - x = Angstroms
             - V = eV
 
-        Thus, to get the first term to eV we need hbar in units of sqrt(eV Angstroms^2 u).
-        Starting with hbar in eVs, we need to convert eV to (Angstroms^2 u / s^2) s.t. the first term comes out with eV.
-        https://www.wolframalpha.com/input/?i=1+%28atomic+mass+units%29*%28angstroms%5E2%29%2F%28s%5E2%29+in+eV
-        Except that didn't seem to work, so for now just use HBAR as defined in the header
+        Thus, to get the first term to eV we need to multiply it by eV s / A**2 / u.
+        The math is here: https://www.wolframalpha.com/input/?i=%28eV*s%29%5E2%2F%28AMU%29+%2F%28Angstrom%5E2%29+in+eV
         """
-        return -(HBAR**2 / (2 * self.input.mass)) * self.mesh.laplacian(psi) + self._potential_psi(psi)
+        return -(HBAR ** 2 / (2 * self.input.mass) * EV_S_PER_ANG_SQUARED_PER_AMU_IN_EV) * self.mesh.laplacian(psi) + \
```

Comment:
```suggestion
        return -(HBAR ** 2 / (2 * self.input.mass)) * self.mesh.laplacian(psi) * EV_S_PER_ANG_SQUARED_PER_AMU_IN_EV + \
```

This is a deep nit. I just don't like the unit conversion factor breaking up all the things that have units (the laplacian is what gives us our per ang^2)